examples/social/README.md

```diff
@@ -0,0 +1,133 @@
+# A Social Media Example Application
+
+This example illustrates how to use channels for cross-chain messages.
+
+For simplicity, each microchain represents one user—its owner. They can subscribe to other
+users and make text posts that get sent to their subscribers.
+
+## How it Works
+
+The application's state on every microchain contains a set of posts created by this chain
+owner, and a set of posts received from other chains that it has subscribed to. The
+received posts are indexed by timestamp, sender and index.
+
+There are `Subscribe` and `Unsubscribe` operations: If a chain owner includes these in a
+new block, they can subscribe to or unsubscribe from another chain.
+
+There is also a `Post` operation: It creates a new post and sends it to a channel, so that
+it reaches all subscribers.
+
+There are corresponding `RequestSubscribe`, `RequestUnsubscribe` and `Posts` cross-chain
+message variants that are created when these operations are handled. The first two are
+sent directly to the chain we want to subscribe to or unsubscribe from. The latter goes
+to the channel.
+
+## Usage
+
+To try it out, first setup a local network with two wallets, compile the `social` example
+and create an application with it:
+
+```bash
+./scripts/run_local.sh
+
+export LINERA_WALLET="$(realpath target/debug/wallet.json)"
+export LINERA_STORAGE="rocksdb:$(dirname "$LINERA_WALLET")/linera.db"
+export LINERA_WALLET_2="$(realpath target/debug/wallet_2.json)"
+export LINERA_STORAGE_2="rocksdb:$(dirname "$LINERA_WALLET_2")/linera_2.db"
+
+cd examples/social && cargo build --release && cd ..
+
+linera --wallet $LINERA_WALLET --storage $LINERA_STORAGE publish-and-create examples/target/wasm32-unknown-unknown/release/social_{contract,service}.wasm
```

Comment:
Some places use `target/debug/linera` and others use `linera` instead. The former works better (at least in my case), but feels like unnecessary verbosity.

So two things I guess:
1. What should we use for now?
2. Assuming we'll eventually want to only use `linera`, what is the longer term path forward?

Reply:
Right, I was meaning to use `linera`, like in the mdbook, but locally I used `target/debug/linera`. Not sure what's best; maybe for the READMEs we should use `target/debug/linera`—or `cargo run --bin linera`? :confused:

Reply:
Yeah, I think that makes sense. I vote for `cargo run --bin linera -- ...`, at least for now.

Reply:
I think let's stick with `linera` IMO. It now works well stand-alone and our docs should be consistent with our examples.

Reply:
You mean just assume `linera` is in the `$PATH`?